gravitee-am-ui/src/app/domain/settings/users/user/devices/devices.component.ts

```diff
@@ -34,6 +35,7 @@ export class UserDevicesComponent implements OnInit {
   canDelete: boolean;
   devices: any[];
   deviceIdentifiers: any[];
+  sortType = SortType.multi;
```

Comment:
**question**: is it a good practice in UI to do this ? When I read the change in the template, I was thinking there is an error until reading this change :thinking:

Reply:
This is to use the ngx-datatable enum As typescript is stricter, it indicates this error
![image](https://github.com/gravitee-io/gravitee-access-management/assets/4974420/ab9c3772-5bbe-44bc-9b79-92b0b7c1eca6)